### PR TITLE
fix: funcDeclaration body check

### PR DESCRIPTION
**Pull request by rytsh**
**Describe the PR**
Nil pointer check in function scoped types.

**Relation issue**
#1309 

This is fixes problem on running swag build.

### CODE REVIEW

**Review by liut — COMMENTED** [review]
looks good!

**Review by mikolajsemeniuk — APPROVED** [review]

File: packages.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func (pkgDefs *PackagesDefinitions) parseTypesFromFile(astFile *ast.File, packag
 
 func (pkgDefs *PackagesDefinitions) parseFunctionScopedTypesFromFile(astFile *ast.File, packagePath string, parsedSchemas map[*TypeSpecDef]*Schema) {
 	for _, astDeclaration := range astFile.Decls {
-		if funcDeclaration, ok := astDeclaration.(*ast.FuncDecl); ok {
+		if funcDeclaration, ok := astDeclaration.(*ast.FuncDecl); ok && funcDeclaration.Body != nil {
 			for _, stmt := range funcDeclaration.Body.List {
```
Comment (ubogdan) [inline packages.go:168]: It's more readable with 

```go
funcDeclaration, ok := astDeclaration.(*ast.FuncDecl);
if  ok && funcDeclaration.Body != nil {
```

Comment (rytsh) [reply packages.go:168]: Hmm, I checked other functions some of areas used this way some not. I can change it.

Comment (rytsh) [reply packages.go:169]: Changed ✔️

**Comment by mstrYoda** [comment]
LGTM

Thanks for the fix 🙏

**Review by ubogdan — APPROVED** [review]
LGTM

**Comment by ubogdan** [comment]
@rytsh Thanks for your contribution

**Comment by morremeyer** [comment]
Thanks for the fast fix, everyone! Can someone let us know when this will be released?

It seems that releases are usually done monthly, not when changes are merged, but I could not find any documentation about it.

Would be happy about knowing more here, thank you al!

**Comment by ubogdan** [comment]
Well, there is not written anywhere. I'm currently doing a release every month with some delay when too few contributions are available.

**Comment by Mokto** [comment]
Given it's a major regression for everyone here, could we please release it ASAP ?

Thanks.

**Comment by mstrYoda** [comment]
> Given it's a major regression for everyone here, could we please release it ASAP ?
> 
> Thanks.

You can use swaggo 1.8.4 version without hassle (actually any version lower than 1.8.5)

**Comment by TomHellier** [comment]
Agree, can we release 1.8.6?

relates to #1327

**Pull request merged**